sdk/servicebus/service-bus/test/common/environmentVariables.ts

```diff
@@ -0,0 +1,216 @@
+  }
+
+  
+
+  // Entity related
+
+  let SERVICEBUS_CONNECTION_STRING = isNode
+    ? process.env.SERVICEBUS_CONNECTION_STRING 
+    // @ts-ignore
+    : window.__env__[Constants.SERVICEBUS_CONNECTION_STRING];
+  if (!SERVICEBUS_CONNECTION_STRING) {
+    throw getError(ErrorCode.MISSING_ENV_VAR, [Constants.SERVICEBUS_CONNECTION_STRING]);
+  }
+
+  let CLEAN_NAMESPACE = isNode
+    ? process.env.CLEAN_NAMESPACE 
+    // @ts-ignore
+    : window.__env__[Constants.CLEAN_NAMESPACE];
+ 
+  let TOPIC_FILTER_NAME = isNode
+    ? process.env.TOPIC_FILTER_NAME 
+    // @ts-ignore
+    : window.__env__[Constants.TOPIC_FILTER_NAME];
+ 
+  let TOPIC_FILTER_SUBSCRIPTION_NAME = isNode
+    ? process.env.TOPIC_FILTER_SUBSCRIPTION_NAME 
+    // @ts-ignore
+    : window.__env__[Constants.TOPIC_FILTER_SUBSCRIPTION_NAME];
+ 
+  let TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME = isNode
+    ? process.env.TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME 
+    // @ts-ignore
+    : window.__env__[Constants.TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME];
+ 
+  let QUEUE_NAME = isNode
+    ? process.env.QUEUE_NAME 
+    // @ts-ignore
+    : window.__env__[Constants.QUEUE_NAME];
+ 
+  let TOPIC_NAME = isNode
+    ? process.env.TOPIC_NAME 
+    // @ts-ignore
+    : window.__env__[Constants.TOPIC_NAME];
+ 
+  let SUBSCRIPTION_NAME = isNode
+    ? process.env.SUBSCRIPTION_NAME 
+    // @ts-ignore
+    : window.__env__[Constants.SUBSCRIPTION_NAME];
+ 
+  let QUEUE_NAME_NO_PARTITION = isNode
+    ? process.env.QUEUE_NAME_NO_PARTITION 
+    // @ts-ignore
+    : window.__env__[Constants.QUEUE_NAME_NO_PARTITION];
+ 
+  let QUEUE_NAME_SESSION = isNode
+    ? process.env.QUEUE_NAME_SESSION 
+    // @ts-ignore
+    : window.__env__[Constants.QUEUE_NAME_SESSION];
+ 
+  let QUEUE_NAME_NO_PARTITION_SESSION = isNode
+    ? process.env.QUEUE_NAME_NO_PARTITION_SESSION 
+    // @ts-ignore
+    : window.__env__[Constants.QUEUE_NAME_NO_PARTITION_SESSION];
+ 
+  let TOPIC_NAME_NO_PARTITION = isNode
+    ? process.env.TOPIC_NAME_NO_PARTITION 
+    // @ts-ignore
+    : window.__env__[Constants.TOPIC_NAME_NO_PARTITION];
+ 
+  let TOPIC_NAME_SESSION = isNode
+    ? process.env.TOPIC_NAME_SESSION 
+    // @ts-ignore
+    : window.__env__[Constants.TOPIC_NAME_SESSION];
+
+  let TOPIC_NAME_NO_PARTITION_SESSION = isNode
+    ? process.env.TOPIC_NAME_NO_PARTITION_SESSION 
+    // @ts-ignore
+    : window.__env__[Constants.TOPIC_NAME_NO_PARTITION_SESSION];
+ 
+  let SUBSCRIPTION_NAME_NO_PARTITION = isNode
+    ? process.env.SUBSCRIPTION_NAME_NO_PARTITION 
+    // @ts-ignore
+    : window.__env__[Constants.SUBSCRIPTION_NAME_NO_PARTITION];
+ 
+  let SUBSCRIPTION_NAME_SESSION = isNode
+    ? process.env.SUBSCRIPTION_NAME_SESSION 
+    // @ts-ignore
+    : window.__env__[Constants.SUBSCRIPTION_NAME_SESSION];
+ 
+  let SUBSCRIPTION_NAME_NO_PARTITION_SESSION = isNode
+    ? process.env.SUBSCRIPTION_NAME_NO_PARTITION_SESSION 
+    // @ts-ignore
+    : window.__env__[Constants.SUBSCRIPTION_NAME_NO_PARTITION_SESSION];
```

Comment:
Would it help to make a function like the following one?:

```typescript
function getEnv(name: string): string | undefined {
  return isNode ? process.env[name] : window.__env__[Constants[name]]
}
```

Feel free to ignore this :sun_with_face:

Reply:
Oh, I wasn't sure if `process.env.X` is same as `process.env["X"]`.
I suppose it is, will try - thank you!